plugins/importexport/native/pkp-native.xsd

```diff
@@ -267,10 +267,12 @@
 		</sequence>
 		<attribute name="date_submitted" type="date" use="optional" />
 		<attribute name="date_published" type="date" use="optional" />
-		<attribute name="version" type="int" use="optional" />
 		<attribute name="status" type="int" use="optional" />
 		<attribute name="primary_contact_id" type="int" use="optional" />
 		<attribute name="url_path" type="string" use="optional" />
+		<attribute name="version_stage" type="string" use="optional" />
+		<attribute name="version_minor" type="string" use="optional" />
```

Comment:
The major and minor should be `int`, no?

Reply:
Yes - changed to int